lib/msal-common/src/authority/Authority.ts

```diff
@@ -172,6 +176,15 @@ export class Authority {
         }
     }
 
+    /**
+     * Region used if any
+     * 
+     * @returns string
+     */
+    public getRegionDiscoveryMetadata(): RegionDiscoveryMetadata {
+        return this.regionDiscoveryMetadata;
```

Comment:
Can you make `this.regionDiscoveryMetadata` public instead of adding a getter function?

Reply:
Yes, refactored it as suggested.